### ISSUE TITLE
Schema load error when upgrading from 5.1.3 -> 5.2.0 (and higher)

### ISSUE THREAD

**Issue by saurori** [gobuffalo/pop#579]
## Description

After upgrading past `v5.1.3`, CI tests are failing to load the DB schema. Locally on Mac OS, the schema loads fine (via `buffalo test`). However on CircleCI, the schema fails to load with:

```
[POP] 2020/08/03 01:27:14 info - dropped database example_test
[POP] 2020/08/03 01:27:14 info - created database example_test
Usage:
  buffalo test [flags]

Flags:
  -h, --help   help for test

ERRO[0000] Error: unable to load schema for example_test: sql: unknown driver "postgres" (forgotten import?) 

Exited with code exit status 255
```

### Steps to Reproduce the Problem

  1. Update to pop 5.2.3
  1. Run `buffalo test` on CircleCI

### Expected Behavior

Tests pass

### Actual Behavior

Tests fail with error loading schema.

### Info

CircleCI images:

```
    docker:
      - image: circleci/golang:1.14-node
      - image: circleci/postgres:12.1-alpine
```

**Comment by stanislas-m** [comment]
Hello, can you give me the contents of your `database.yml` file? Thanks!

**Comment by saurori** [comment]
CircleCI `database.yml`:
```
---
test:
  dialect: postgres
  database: example_test
  user: postgres
  host: 127.0.0.1
  pool: 5
```
Local `database.yml` (test portion):
```
test:
  url: {{envOr "TEST_DATABASE_URL" "postgres://postgres@127.0.0.1:5432/example_test?sslmode=disable"}}
```

**Comment by stanislas-m** [comment]
@saurori I forgot to release buffalo-pop plugin with this version, can you update and try again?

**Comment by saurori** [comment]
Getting the same error as above. Here is the `go mod download` step of the CI showing the new deps:
```
go: github.com/jstemmer/go-junit-report upgrade => v0.9.1
go: found github.com/gobuffalo/buffalo/buffalo in github.com/gobuffalo/buffalo v0.16.14
go: downloading github.com/gobuffalo/buffalo-pop/v2 v2.1.0
go: downloading github.com/gobuffalo/pop/v5 v5.2.3
go: downloading github.com/jackc/pgconn v1.6.0
go: downloading github.com/jackc/pgx v3.6.2+incompatible
go: downloading github.com/jackc/pgservicefile v0.0.0-20200307190119-3430c5407db8
go: downloading github.com/jackc/pgproto3/v2 v2.0.2
go: downloading github.com/jackc/pgx/v4 v4.6.0
go: downloading github.com/jackc/pgtype v1.3.0
```

**Comment by stanislas-m** [comment]
OK, I think I finally found the issue. It should be fixed with #585.

**Comment by saurori** [comment]
Updated to 5.2.4 and everything is working now, schema loads and tests pass. Thanks!
